Mathlib/RingTheory/LocalProperties/Basic.lean

```diff
@@ -110,14 +118,26 @@ def RingHom.HoldsForLocalizationAway : Prop :=
   ∀ ⦃R : Type u⦄ (S : Type u) [CommRing R] [CommRing S] [Algebra R S] (r : R)
     [IsLocalization.Away r S], P (algebraMap R S)
 
+/-- A property `P` of ring homs satisfies `RingHom.StableUnderCompositionWithLocalizationAwayLeft`
+if whenever `P` holds for `f` it also holds for the composition with
+localization maps on the left. -/
+def RingHom.StableUnderCompositionWithLocalizationAwayLeft : Prop :=
+  ∀ ⦃R : Type u⦄ (S : Type u) ⦃T : Type u⦄ [CommRing R] [CommRing S] [CommRing T] [Algebra R S]
+    (r : R) [IsLocalization.Away r S] (f : S →+* T), P f → P (f.comp (algebraMap R S))
```

Comment:
Is this the right naming? In the `RingHom.comp` sense, we compose on the right here.

Reply:
Yeah I am taking the `>>` direction.

Reply:
What about `StableUnderCompositionWithLocalizationAwaySource` vs `StableUnderCompositionWithLocalizationAwayTarget`? (In alignment with the `OfLocalizationSpan` terminology)?

Reply:
Sure! Renamed.